packages/cli/create-strapi-starter/src/create-strapi-starter.ts

```diff
@@ -31,7 +31,6 @@ program
   .version(packageJson.version)
   .arguments('[directory], [starter]')
   .option('--use-npm', 'Force usage of npm instead of yarn to create the project')
-  .option('--debug', 'Display database connection error')
```

Comment:
I assume a database error would still be displayed without the debug tag right? But the debug logs would not appear.

Reply:
we removed the db connection test a long time ago this is just dead code it's not doign anythign :)